### PR TITLE
Enable whitespace linter & fix findings

### PR DESCRIPTION
**Pull request by jtraglia**
**What type of PR is this?**

> Other

**What does this PR do? Why is it needed?**

Removes all of the unnecessary whitespace. Enables the `whitespace` linter to catch this in the future.

### CODE REVIEW

File: math/math_helper.go [hunk 1]
```diff
@@ -188,7 +188,7 @@ func Mod64(a, b uint64) (uint64, error) {
 	return val, nil
 }
 
-// Int returns the integer value of the uint64 argument. If there is an overlow, then an error is
+// Int returns the integer value of the uint64 argument. If there is an overflow, then an error is
```
Comment (jtraglia) [inline math/math_helper.go:191]: Also, fixes this one typo I noticed.

**Review by terencechain — DISMISSED** [review]

**Review by rkapka — DISMISSED** [review]

**Comment by jtraglia** [comment]
> Hi @jtraglia , the build fails at the moment.

Fixed 😄

**Review by prestonvanloon — DISMISSED** [review]
LGTM -- Nice addition thanks!

**Comment by rkapka** [comment]
> > Hi @jtraglia , the build fails at the moment.
> 
> Fixed smile

It has conflicts now @jtraglia :(

**Comment by prestonvanloon** [comment]
I fixed the conflicts. Not sure why github was too dumb to figure out that a file was renamed in whole. There was no conflict resolution required locally /shrug

**Review by prestonvanloon — APPROVED** [review]

**Pull request merged**